api/v1beta1/openstackbaremetalset_types.go

```diff
@@ -88,7 +88,6 @@ type OpenStackBaremetalSetSpec struct {
 	// +kubebuilder:validation:Optional
 	CtlplaneGateway string `json:"ctlplaneGateway,omitempty"`
 	// +kubebuilder:validation:Optional
-	// +kubebuilder:default="255.255.255.0"
```

Comment:
We should remove this from API as it's not used anymore. Was left behind[1] to not break any usage. I think we can safely remove this.
[1] https://github.com/openstack-k8s-operators/openstack-baremetal-operator/blob/main/pkg/openstackbaremetalset/baremetalhost.go#L102

Reply:
ack, let's remove it